### PR TITLE
Added calibration data for Tamron E 17-28mm and E 28-200mm

### LINKED ISSUE #1395: Add Tamron 17-28 mm F2.8 Di III RXD Sony E
Please add Tamron 17-28 mm F2.8 Di III RXD Sony E
Thank you

### LINKED ISSUE #1430: Calibration data for Tamron E 17-28mm F2.8-2.8
Please add the attached lens data to the lensfun data base.
[Tamron E 17-28mm F2.8-2.8.xml.zip](https://github.com/lensfun/lensfun/files/6604638/Tamron.E.17-28mm.F2.8-2.8.xml.zip)

### LINKED ISSUE #1431: Calibration data for Tamron E 28-200mm F2.8-5.6 A071
Please add the attached lens data to the lensfun data base.
[Tamron E 28-200mm F2.8-5.6 A071.xml.zip](https://github.com/lensfun/lensfun/files/6604640/Tamron.E.28-200mm.F2.8-5.6.A071.xml.zip)

### PR DESCRIPTION
**Pull request by axeleroy**
Thanks to @fheinig for the calibration data.

Closes #1395, closes #1430, closes #1431

PS: I made a single PR for both lenses, I do not know if I should have done two separate PRs.

### CODE REVIEW

File: data/db/mil-tamron.xml [hunk 1]
```diff
@@ -224,5 +224,56 @@
...
+	        <distortion model="ptlens" focal="26" a="0.00474933" b="0.00263057" c="-0.0114455"/>
+	        <distortion model="ptlens" focal="28" a="0.0105916" b="-0.0179303" c="0.0119596"/>
+        </calibration>
+    </lens>
+    
+    <lens>
+        <maker>Tamron</maker>
+        <model>E 28-200mm F2.8-5.6 A071</model>
+        <mount>Sony E</mount>
+	      <aperture max="5.6" min="2.8"/>
```
Comment (gnl21) [inline data/db/mil-tamron.xml:253]: Some of these lines have got weird tab alignments. Would you mind sorting them out?

**Comment by gnl21** [comment]
> PS: I made a single PR for both lenses, I do not know if I should have done two separate PRs.

Both in one is fine, it can just hold things up if there's a problem with one but not the other. (And at the moment a lack of maintainers means things take long enough either way!)

Thanks to both of you for doing this. Making things into PRs does help things get merged faster because I don't currently have the time to go through the issue to identify the ones that already have profile data in them.

**Comment by axeleroy** [comment]
I fixed the tab alignment issue in a second commit, should I squash my commits right now or they will be when merged?

**Comment by gnl21** [comment]
Perfect, thanks. I'll merge this PR when the CI passes then.

If you want the commits squashed then I can squash them when I merge, although I usually wouldn't bother. I'm fine with the branch being merged in as is.

**Comment by axeleroy** [comment]
Thanks a lot! 

Speaking of going through issues, I believe we can close #1423 as well, it refers to calibration images for the 17-28mm.

**Comment by gnl21** [comment]
Yes, it looks like it. I'll post a comment asking the uploader to have a check of these profiles when they're merged and close the issue.

By the way, the issues that you've mentioned above don't seem to be linked to the PR, possibly because you've put "Closes issues" but GitHub expects just "Closes".

**Comment by axeleroy** [comment]
Right, and it's extremely picky if I want to link multiple issues…

**Comment by gnl21** [comment]
That's great, thanks, but (sorry about this) can I now ask you to unlink #1423? I've realised that it might contain vignetting data that isn't in this PR and I don't have any way to contact the uploader except to let @bronger's script close the issue and send a comment to the uploader. (This is because I don't have access to the calibration data uploads at the moment and I haven't been able to contact @bronger to sort it out).

**Comment by axeleroy** [comment]
No problem, I unlinked it after your last comment.

**Comment by gnl21** [comment]
Thanks for all the updates, @axelroy.

**Pull request merged**